<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnlimitedDialog</class>
 <widget class="QDialog" name="UnlimitedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unlimited</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMining">
      <attribute name="title">
       <string>&amp;Mining</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_Main">
       <item>
        <layout class="QHBoxLayout" name="maxgenblockszLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
             <string>The largest block that will be mined</string>
           </property>
           <property name="text">
            <string>Maximum Generated Block Size (bytes) </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="miningMaxBlock">
           <property name="toolTip">
             <string>The largest block that will be mined</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_Main">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNetwork">
      <attribute name="title">
       <string>&amp;Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_Network">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3_Network"/>
       </item>
       <item>
        <widget class="QLabel" name="bandwidth">
         <property name="text">
          <string>Bandwidth Restrictions in KBytes/sec (check to enable):</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="sendShaping">
         <item>
          <widget class="QCheckBox" name="sendShapingEnable">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sendburstlabel">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sendBurstEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sendBurstSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sendlabel">
           <property name="text">
            <string>Average</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sendAveEdit">
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sendAveSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="recvShaping">
         <item>
          <widget class="QCheckBox" name="recvShapingEnable">
           <property name="text">
            <string>Receive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="recvburstlabel">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="recvBurstEdit">
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="recvBurstSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="receivelabel">
           <property name="text">
            <string>Average</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="recvAveEdit">
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="recvAveSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="errorText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Block Acceptance</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Excessive Block Size (in bytes)</string>
           </property>
            <property name="toolTip">
              <string>Blocks above this size will be considered &quot;excessive&quot;.  That is, they will not be relayed or become part of the active chain until &quot;Excessive Acceptance&quot; blocks have been mined on top of them.</string>
            </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="excessiveBlockSize">
           <property name="toolTip">
             <string>Blocks above this size will be considered &quot;excessive&quot;.  That is, they will not be relayed or become part of the active chain until &quot;Excessive Acceptance&quot; blocks have been mined on top of them.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="toolTip">
             <string>An excessive block normally blocks a chain from being accepted.  However, if the chain grows deeper than this accept the excessive block (and the chain).</string>
           </property>
           <property name="text">
            <string>Excessive Acceptance Depth (in blocks)</string>
           </property>
          </widget>
         </item>
         <item>
           <widget class="QLineEdit" name="excessiveAcceptDepth">
             <property name="toolTip">
               <string>An excessive block normally blocks a chain from being accepted.  However, if the chain grows deeper than this accept the excessive block (and the chain).</string>
             </property>
           </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_Network">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_Bottom">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Bottom">
        <item>
         <widget class="QLabel" name="overriddenByCommandLineInfoLabel">
          <property name="text">
           <string>Active command-line options that override above options:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_Bottom">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="overriddenByCommandLineLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="toolTip">
        <string>Reset all client options to default.</string>
       </property>
       <property name="text">
        <string>&amp;Reset Options</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
